woodwork/tests/conftest.py

```diff
@@ -50,24 +50,33 @@ def sample_df_pandas():
     return pd.DataFrame(
         {
             "id": range(4),
-            "full_name": [
-                "Mr. John Doe",
-                "Doe, Mrs. Jane",
-                "James Brown",
-                "Ms. Paige Turner",
-            ],
-            "email": [
-                "john.smith@example.com",
-                np.nan,
-                "team@featuretools.com",
-                "junk@example.com",
-            ],
-            "phone_number": [
-                "5555555555",
-                "555-555-5555",
-                "1-(555)-555-5555",
-                "555-555-5555",
-            ],
+            "full_name": pd.Series(
```

Comment:
Just curious why you are specifying the dtype for these columns now. Did things break otherwise?

Reply:
Great point! I changed this during testing earlier, but it works without the casting! Removing now.